### ISSUE TITLE
Series of Questions to help based noob contributer

### ISSUE THREAD

**Issue by Yamnasm** [syssykat/talk-to-humans#7]
`if __name__ == "__main__":
    ~~~~`
What does this do? Why is it necessary?
________________________________________________________________________________________
 `super().~~~`
What does this do?
________________________________________________________________________________________
Could the Server + Channel (feedback, report) IDs be used like this:
```
self.T2H_SERVER = settings["Server_ID"]

self.FEEDBACK_CHANNEL = settings["Feed_ID"]

self.REPORT_CHANNEL = settings["Report_ID"]
```
Purely for keeping all those unique-type informations in one place, in Settings.json

________________________________________________________________________________________
Could this:
```
self.MINUTE = 60
self.HOUR = self.MINUTE * 60
self.DAY = self.HOUR * 24
```

be changed to this:
```
self.MINUTE = 60
self.HOUR = 3600
self.DAY = 86400
```
Is it not quicker to do the latter, memory-wise (it might be insignificant, but I'm purely interested)
________________________________________________________________________________________
`if message.author.bot or message.author == self.user:`
Is this not the same thing? All self.messages surely would be from an author.bot?

**Comment by syssykat** [comment]
Alright let me explain one by one.

First off the if-statement with `__name__` in it is the starting endpoint of the program. Python has a variable called `__name__` (dunder name) which will contain the name of the starting execution. "__main__" is our file and so we check if `__name__` is equal to "__main__" to see if the execution was from this script.

It's like the `int main()` of python.

Second one is called super. What it does is replaces super() with the parent or child inherited class. So in this case it's grabbing the Discord.py Bot class. The `__init__` after it is the constructor which we're firing off during construction of our class.

Next, the channel IDs need to go through the `get_server().get_channel()` statement but yes the raw IDs could actually be stored in the settings.json which I might actually do now.

Also the MINUTE, HOUR, and DAY constants could also be represented like that but the reason I did it the way it is is because it shows the calculation. It's basically just easier to understand. Maybe it's memory-inefficient but it's basically not even noticeable if it is. Still worth checking out just out of curiosity.

Last thing, the if statement you supplied won't work. Python doesn't do `X or Y == Z`. Instead actually it does `X == Z or Y == Z` so it would actually be:
 `if message.author.bot == self.user or message.author == self.user`

As for why it's `message.author.bot` I don't know, I just saw it in other production code.
